src/Compilers/Test/Resources/Core/NetFX/Minimal/mincorlib.cs

```diff
@@ -2,8 +2,8 @@
 
 namespace System
 {
-	public class Object {}
-    public struct Byte {}
+    public class Object { }
```

Comment:
My concern is that this change might invalidate existing tests, i.e. they will stop testing target scenarios because things are no longer missing.